### ISSUE TITLE
Review pages accessibility audit

### ISSUE THREAD

**Issue by pacerwow** [department-of-veterans-affairs/va.gov-team#94614]
## Issue Description
Conduct an accessibility audit of the following pages:

review-and-submit
confirmation

---
## Tasks
- [x] Conduct the accessibility audit as outlined in the Epic for the pages listed above
- [x] Record bugs & violations in Google Sheets

## Acceptance Criteria
- [x] An accessibility audit is performed for the pages listed above
- [x] Results are documented in Google Sheets
- [ ] A summary is shared with our stakeholders during Sprint Review

**Comment by mayacarrolluxa6** [comment]
@pacerwow @bacitracin Why are we auditing the review and confirmation pages? Which OCTO objective is this aligned with? This ticket is going to roll over since the work doesn’t seem to have started. Do we still need to proceed with doing this work?

**Comment by bacitracin** [comment]
The [Accessibility Audit issue tracker ](https://docs.google.com/spreadsheets/d/1paNxD798NSMrjZeUe9WwVQ4xRdl439ypKyPDPW3CXIo/edit?gid=2034611899#gid=2034611899) Google Sheet lives in the engineering folder of the team's Google Drive. If I review any more page for accessibility, I will update the spreadsheet.

18 issues were identified & created as GitHub issues under the [parent epic](https://github.com/department-of-veterans-affairs/va.gov-team/issues/53414). Since I was auditing at a stricter standard, many of the created issues are enhancements.
